packages/relay/src/lib/services/hbarLimitService/index.ts

```diff
@@ -99,12 +99,11 @@ export class HbarLimitService implements IHbarLimitService {
   ) {
     this.reset = this.getResetTimestamp();
 
-    const operatorId = ConfigService.get('OPERATOR_ID_MAIN');
-    const operatorKey = ConfigService.get('OPERATOR_KEY_MAIN');
-    if (operatorId) {
-      this.setOperatorAddress(AccountId.fromString(operatorId as string).toSolidityAddress());
-    } else if (operatorKey) {
-      this.setOperatorAddress(Utils.createPrivateKeyBasedOnFormat(operatorKey as string).publicKey.toEvmAddress());
+    const operator = Utils.getOperator(logger);
+    if (operator) {
+      this.operatorAddress = prepend0x(AccountId.fromString(operator.accountId.toString()).toSolidityAddress());
+    } else {
+      this.operatorAddress = zeroAddress();
```

Comment:
QQ: Why do we want to set it to zero here instead of a more critical option? I understand this is a sub-flow which shouldn't be stopping the requests but could this be misleading if we aim to track the operator's balance through the cache or Prometheus metrics?